configs/envoyproxy_io_proxy_http3_downstream.yaml

```diff
@@ -6,6 +6,51 @@ admin:
       address: 0.0.0.0
       port_value: 9901
 static_resources:
+  listeners:
+    - name: listener_tcp
+      address:
+        socket_address:
+          protocol: TCP
+          address: 0.0.0.0
+          port_value: 10000
+      filter_chains:
+        transport_socket:
```

Comment:
this needs to be a list - ie just add `-` before

Reply:
annoyingly, this loads just fine in Envoy, but the precheck validator is apparently more strict than Envoy.
h/t @yuval-k who called out the same thing over on https://github.com/envoyproxy/envoy/pull/15987

Reply:
yep - it  uses the static validation in the prechecks - and yep its a bit stricter in a few ways - not sure how we can integrate/sync these two tools better - they are both useful checks

Reply:
in respect of this specific problem - i think the check is good - ie we should be explicit with arrays - apart from anything else it makes the config examples in docs much less confusing